client/src/layouts/CompetePageLayout.tsx

```diff
@@ -0,0 +1,34 @@
+import { Grid } from "@mui/material";
+import { ReactNode } from "react";
+
+type Props = {
+  leftGridChildren: ReactNode;
+  rightGridChildren: ReactNode;
+};
+
+const CompetePageLayout = ({
+  leftGridChildren,
+  rightGridChildren,
+}: Props) => {
+  return (
+    <Grid container bgcolor={"background.default"}>
+      <Grid item xs={9} height={"100vh"} width={"100vw"}>
```

Comment:
vh and vw are related to the viewport which we don't want. We can use 100% and we don't need width as we use the grid system and we define the ratio of 9:3